packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -68,14 +69,16 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
                 }
 
                 return pluginUtils.addExtension(normalizedPath, ext);
-            } else {
+            } else if (importer) {
                 try {
-                    return lwcResolver.resolveModule(importee, importer, {
+                    return resolveModule(importee, importer, {
                         modules: customResolvedModules,
                         rootDir: customRootDir,
                     }).entry;
-                } catch {
-                    // noop: let rollup plugins keep resolving
+                } catch (err) {
+                    if (err.code !== 'NO_LWC_MODULE_FOUND') {
```

Comment:
Do you need the `.code` property? Could you not check `err instanceof NoLwcModuleFound`

Reply:
The idea here is to not have to expose the Error constructors but just the code message.